sys/stdio_rtt/stdio_rtt.c

```diff
@@ -319,7 +317,7 @@ static ssize_t rtt_stdio_vfs_write(vfs_file_t *filp, const void *src, size_t nby
 
 #endif
 
-void uart_stdio_init(void) {
+void stdio_init(void) {
     #ifndef RTT_STDIO_DISABLE_STDIN
```

Comment:
Similar to `STDIO_UART_DEV`, shouldn't this then be named `STDIO_RTT_DISABLE_STDIN`?